### DISCUSSION TITLE
Tree view / grid

### DISCUSSION THREAD

**Discussion by Josh-Nicholson** [skeletonlabs/skeleton#1217] [Ideas]
Are there any plans in the existing roadmap to have a tree view/grid component?
If not, does anyone know of any alternatives that I could use?

In the the past I have used [fancytree](https://github.com/mar10/fancytree) which for me is the gold standard of tree components (ignoring the jquery of course).

**Comment by Sarenor** [comment]
We're currently in the process of deciding on a roadmap for after v1.
During the sprint to reach that milestone we didn't really think past it and are now in the process of assessing with what and how we want to continue.

Looking at Requested Features, prioritizing them and - maybe - implementing them is certainly part of that, so a feature Request for a Tree view might be in the cards.

Not sure if it'll turn out like fancytree though, we usually try to provide smaller components that you can put together yourself (so inline editing might not be part of the default, for example).

Also, just because there is a feature request it's not guaranteed that we'll implement it

It's also a rather specific feature that seems like it needs a lot of time and attention.
We may treat it like data tables and just try to point you to a library that implements it and is compatible and working well with Skeleton.

**Comment by endigo9740** [comment]
Feature requests are always welcome. I'd recommend creating a ticket for one. If the core team can get to it, another contributor might. Just be sure to adequately explain what it is and how you expect it to operate. I know what a tree view is, but that link doesn't provide a great example of what this is.

Here's another off the top of my head that might work better:
https://material.angular.io/cdk/tree/overview

**Reply by Josh-Nicholson** [reply]
Thanks for the response.
Here is a better example of what I'm after [Webix Tree](https://webix.com/widget/tree/).
